packages/sanity/src/structure/getIntentState.ts

```diff
@@ -47,12 +47,11 @@ export function getIntentState(
         pane.schemaTypeName === params.type &&
         pane.options.filter === '_type == $type')
     ) {
+      const { template, version } = params;
       const paneParams: {
```

Comment:
Our linter isn't a fan of nested ternaries, and the formatting is slightly off

```suggestion
      const paneParams: {
        template?: string
        version?: string
      } =
        isTemplate || isVersion
          ? {
              ...(isTemplate ? {template: params.template} : {}),
              ...(isVersion ? {version: params.isVersion} : {}),
            }
          : EMPTY_PARAMS
```

Reply:
What's the point of `EMPTY_PARAMS`? I'm guessing it's to reduce the amount of nursery garbage collection, by reusing a fixed object? But this change will create two objects that are immediately garbage collected.

Reply:
It's mostly about keeping referential identity if/when objects are passed on to the React layer (not entirely sure if that's relevant here though) and less about about garbage collection.